akka-persistence/src/main/scala/akka/persistence/journal/PersistencePluginProxy.scala

```diff
@@ -136,8 +131,57 @@ final class JournalProxy(config: Config) extends Actor with Stash with ActorLogg
       if (targetAtThisNode && address != selfAddress)
         becomeIdentifying(address)
     case InitTimeout ⇒
-    case msg ⇒
-      targetJournal.forward(msg)
+
+    case msg @ WriteMessages(messages, _, actorInstanceId) ⇒
+      implicit val ec = context.system.dispatcher
+      val sender = context.sender()
+      val f = ask(targetJournal, msg)(1.second)
+      f onComplete {
```

Comment:
From what I understand, pipeTo only returns 1 message whereas when this future fails we may need to send the actor several messages. I'm open to suggestions on how to make it cleaner, though.